source/bdDetect.py

```diff
@@ -256,13 +258,13 @@ def _bgScan(self, param):
 						with self._btDevsLock:
 							self._btDevs = btDevsCache
 
-	def rescan(self):
+	def rescan(self, usb=True, bluetooth=True, limitToDevices=None):

```

Comment:
I wonder whether this is now doing what you intend it to do. Note that window messages also trigger a rescan, and as far as I can see, these window messages now still trigger a full rescan instead. This might also obfuscate the problem I reported to you on Twitter earlier about resumeDetection being called after a bluetooth device is disconnected while a background scan is running.

Reply:
Thanks for the heads up. I'll have to look at the bdDetect code some more. It may be that window messages are not only triggering a scan for both BT/USB, but also a rescan for more than just the current driver, which is not what we want here. Since you're quite familiar with bdDetect, do you have any concrete suggestions to improve this?